spec/newspaper_works_rake_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'spec_helper'
+require 'rake'
+require 'ndnp_shared'
+require 'active_fedora/cleaner'
+
+# rubocop:disable RSpec/DescribeClass
+describe 'newspaper_works rake tasks' do
+  include_context 'ndnp fixture setup'
+
+  before(:all) do
+    # task_path = File.join(NewspaperWorks::GEM_PATH, 'lib/tasks/newspaper_works_tasks')
```

Comment:
Remove commented line (unless needed for future reference?)

Reply:
Removed in f40bf67762537a0abd84dbeadbb11144cf9972d6